web-console/1-homepage/catalog.md

```diff
@@ -33,7 +35,16 @@
 
 ![Import](img/Import.png)
 
-- **Note:** Both Deploy Image and Import YAML are available options from inside the console as well, but when launching either from the homepage, there will include a "Add to Project" selection at the top to determine where to add the item.
+- Select from project will bring down an overlay panel with the option to select a project
+
+![Select](img/Select-02.png)
+
+- Once a project has been selected, the user can choose a builder image or template from that project to order as usual.
+
+![Select](img/Select-04.png)
+
+
+- **Note:** Deploy Image,  Import YAML, and Select from Project are available options from inside the console as well, but when launched from there, they do not include an "Add to Project" field at the top of the form.
```

Comment:
@cshinn in this Note, we may want to be more specific and state that you are talking about the Configuration step.  What do you think?